data/src/main/java/com/linkedin/data/codec/ProtobufDataCodec.java

```diff
@@ -57,29 +56,27 @@
   private static final byte RAW_BYTES_ORDINAL = 10;
   private static final byte NULL_ORDINAL = 11;
 
-  private static final Map<Class<?>, Byte> ORDINAL_MAP = new HashMap<>();
-  static {
-    ORDINAL_MAP.put(DataMap.class, MAP_ORDINAL);
-    ORDINAL_MAP.put(DataList.class, LIST_ORDINAL);
-    ORDINAL_MAP.put(String.class, STRING_LITERAL_ORDINAL);
-    ORDINAL_MAP.put(Integer.class, INTEGER_ORDINAL);
-    ORDINAL_MAP.put(Long.class, LONG_ORDINAL);
-    ORDINAL_MAP.put(Float.class, FLOAT_ORDINAL);
-    ORDINAL_MAP.put(Double.class,DOUBLE_ORDINAL);
-    ORDINAL_MAP.put(Boolean.class, BOOLEAN_FALSE_ORDINAL);
-    ORDINAL_MAP.put(ByteString.class, RAW_BYTES_ORDINAL);
-  }
+  // 12-19 is reserved for custom ordinals to be used by codec extenders.
+  private static final byte ASCII_STRING_LITERAL_ORDINAL = 20;
```

Comment:
curious why ASCII ordinal has to be 20?

Reply:
We reserved 12-19 for custom ordinals to be used by codec extenders. Protobuf codec for example is extended within LI and used in dedupe mode for frontends at LinkedIn.